index.js

```diff
@@ -21,3 +21,39 @@ getJoke().then(joke => {
 
     document.getElementById('fact').innerHTML = value
 })
+
+/* Enter Website */
+const pages = document.getElementsByTagName('main')
+const enter = document.getElementById('enter')
+
+enter.addEventListener('click', togglePage)
+
+function togglePage() {
+    pages[0].classList.toggle('hidden') /* hide home page */
+    pages[1].classList.toggle('hidden') /* show categories page */
+}
+
+/* Get Fact by Category */
+const buttons = document.getElementsByClassName("category-button");
+
+for (button of buttons) {
+  button.addEventListener("click", (e) => {
+    pages[1].classList.toggle('hidden') /* hide categories page */
+    pages[2].classList.toggle('hidden') /* show fact page */
+
+    input = e.target.innerText.toLowerCase(); /* select button text, convert to lowercase*/
+    getJoke(input).then(joke => {
+        const {categories, icon_url, id, url, value} = joke
+        document.getElementById('category-fact').innerHTML = value
```

Comment:
When a new joke is shorter than the previous joke, part of the old joke is still visible, though this surprises me.  Here are screen shots.

A long joke:
![Screen Shot 2023-01-19 at 8 43 33 PM](https://user-images.githubusercontent.com/51532684/213619775-9dc77695-9ce7-4e9f-8e7e-38e2a6a40fb2.png)

The next short joke:
![Screen Shot 2023-01-19 at 8 43 43 PM](https://user-images.githubusercontent.com/51532684/213619867-232aad40-98ca-4b69-b3e7-1e3b1be02037.png)

Reply:
I think its due to the 'autograph' `position: absolute`. It also breaks when we change the width of the window.

Reply:
I don't understand why the position of the autograph would affect "rewriting" the inner-html.  I see now what you mean about the problem with changing the width of the window and what happens with autograph as well as the approval stamp.